test/test_parse_user_storage.py

```diff
@@ -30,11 +30,17 @@ def db():
     # dbfile = "sqlite:///usage.db"
     return ProjectDataset(project,dbfile)
 
-def test_parse_lquota(db):
+def test_parse_user_storage(db):
 
     parse_file_report('test/2022-11-02T11:36:45.w40.scratch.json', verbose=verbose, db=db)
     parse_file_report('test/2022-11-02T11:36:45.w40.gdata.json', verbose=verbose, db=db)
 
+
+def test_parse_user_storage_with_error(db):
+
+    parse_file_report('test/2022-11-02T11:36:45.w40.gdata.bad-json', verbose=verbose, db=db)
```

Comment:
This is just a file that doesn't exist right?

Could `touch` a zero length file to test this also works if the file is there but empty.

Reply:
It's committed as an 'almost empty file', it just has a couple of newlines in it. We could also use touch as you suggest. If the file is missing the behavior of dumpstats (not this function) would be different (it would be better).